source/docs/basics-of-scripting/memory-flags-and-definitions.rst

```diff
@@ -220,7 +220,72 @@ Of course, you can do a bit more with flags than you can with definitions.
 2B. Math With Flags
 ~~~~~~~~~~~~~~~~~~~
 
+You can do more than just store raw information with flags - you can also do math! Flag math operations support addition, subtraction, multiplication, and division. The operation you specify will be performed on the value currently stored. Note that this only works for flags with numerical values - if you have stored text on a flag, you cannot use math to change the value!
 
+For example, first we'll use the ``/ex`` command to store a value in a flag:
+
+.. code::
+
+    /ex flag server DoSomeMath 10
```

Comment:
Did you mean to put a space between `DoSomeMath` and `10`?

Also, generally speaking since most things involving an assigned ID tend to be case insensitive, I think it's a good idea to use all lowercase names, plus underscores.

Reply:
I mean, yes insofar as I am more used to using `define` than `flag` and I probably should have checked before committing.

I use CapitalCase or whatever in my internal scripts for clarity, but if you want all-lowercase that's fine!

Reply:
For internal/personal use, CapitalCase is fine. It's just that for a public script or something, it should be encouraged to not use CaptialCase since names tend to be case insensitive anyways, so it's less of a hassle to just use all lowercase plus underscores. Saves the theoretical trouble of naming two definitions "HelloWorld" and "HELLOworld" and them overriding each other.

Reply:
Sure, that is good to know! I look forward to a design document, and hopefully we can augment it over time by way of me attempting contributions. 😄

Reply:
A technical detail:

Shouldn't you put that command in a script container? I know it may seem redundant, but a task or world script may better frame the command to the user.

Reply:
I thought it was a good example of being able to set flags with the command, but sure that would work too.